### PR TITLE
Site 6640 adds Minimal Clock Icon and Heads Icon

### PR DESCRIPTION
**Pull request by Jiert**

### CODE REVIEW

File: app/components/ClockMinimal.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('ClockMinimal', {
+  className: 'tilt-icon-clock-minimal',
+
+  render() {
+    return (
+      <svg className={this.getClassNames()} viewBox="0 0 23 23" >
+        <title>ClockMinimal</title>
+        <g className="tilt-icon-main" transform="translate(-511.000000, -1908.000000)">
```
Comment (cohitre) [inline app/components/ClockMinimal.js]: you can add up all the translates into a single declaration and get rid of some unnecessary `<g/>`s
```
-511 + 484 + 27 = 0
-1908 + 1901 + 7 = 0
```

in this case we don't even need to declare the transforms.

File: app/components/Heads.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('Heads', {
+  className: 'tilt-icon-heads',
+
+  render() {
+    return (
+      <svg className={this.getClassNames()} viewBox="0 0 20 17">
+        <title>Heads</title>
+        <g className="tilt-icon-main">
```
Comment (cohitre) [inline app/components/Heads.js]: same as above, consolidate all the `transform=translate` into a single `<g/>` (or remove if not needed)

File: app/stylesheets/Free.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.tilt-icon.tilt-icon-free {
+.tilt-icon.tilt-icon-tag-free {
```
Comment (cohitre) [inline app/stylesheets/Free.scss]: is this change needed? we just added the `Free` icon earlier today, but this could potentially be a breaking change.

**Comment by cohitre** [comment]
see comments... lgtm.

File: app/stylesheets/Free.scss (hunk 3 above)
Comment (Jiert) [inline app/stylesheets/Free.scss]: @cohitre Are the tilt-images stylesheets used for anything other than styling the images served at the local express server? This change wires up the Free icon to the styles declared in Free.scss.

image





image

**Comment by Jiert** [comment]
@cohitre @tildedave updates made

File: app/stylesheets/Free.scss (hunk 3 above)
Comment (cohitre) [inline app/stylesheets/Free.scss]: good catch. this was a bug.

**Comment by cohitre** [comment]
lgtm.
